drivers/bus/acpi_new/acpi.rc

```diff
@@ -0,0 +1,5 @@
+#define REACTOS_VERSION_DLL
+#define REACTOS_STR_FILE_DESCRIPTION  "uACPI ReactOS ACPI Driver"
+#define REACTOS_STR_INTERNAL_NAME     "uACPI"
+#define REACTOS_STR_ORIGINAL_FILENAME "uACPI.sys"
```

Comment:
The generated filename is acpi.sys, so... I don't know.

Reply:
shrug